Beta docs (#900)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaAttributeSet.md

### GetById

```powershell
Get-EntraBetaAttributeSet 
 -Id <String> 
 [<CommonParameters>]
```

## Description

Gets a list of Microsoft Entra ID attribute sets. Specify `Id` parameter to retrieve an attribute set.

## Examples

### Example 1: Get an all attribute sets

```powershell
Connect-Entra -Scopes 'CustomSecAttributeDefinition.ReadWrite.All'

> Set-EntraBetaAttributeSet 

Reviewer: ```suggestion
Get-EntraBetaAttributeSet
```

Reviewer 2: Changes done

```

```Output
Id                    Description                           MaxAttributesPerSet
--                    -----------                           -------------------
engineering           Attributes for cloud engineering team 25
Test2                 Attributes for test2  team            25
demo                  NEW Demo Description                  25
demo2                 TEST THE DEMO
DEMO5
DEMO6                 Description to test the demo6
DEMO7                 Description to test the demo6         24

> demo3                 Demo3's Description                   33

Reviewer 2: changes done

```

This example returns all attribute sets.

### Example 2: Get an attribute sets

```powershell
Connect-Entra -Scopes 'CustomSecAttributeDefinition.ReadWrite.All'

> Get-EntraAttributeSet -Id 'Testing'

Reviewer: ```suggestion
Get-EntraBetaAttributeSet -Id 'Testing'
```

Reviewer 2: Changes Done

```

```Output
Id      Description                     MaxAttributesPerSet
--      -----------                     -------------------
Testing Attributes for engineering team 10
```

This example demonstrates how to retrieve an attribute set by Id.

> - `Id` parameter specifies the Id of a Microsoft Entra ID object.

Reviewer: ```suggestion
- `Id` parameter specifies the unique identifier for the attribute set within a tenant.
```

Reviewer 2: changes done

## Parameters

### -Id

> The unique identifier of a Microsoft Entra ID set object.

Reviewer: ```suggestion
Unique identifier for the attribute set within a tenant. This identifier can be up to 32 characters long and may include Unicode characters. It cannot contain spaces or special characters, and it cannot be changed later. The identifier is case insensitive.
```

Reviewer 2: Changes done
